<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDeviceDialog</class>
 <widget class="QWidget" name="AddDeviceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add device</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="SlidingStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="PageConnectionType">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;h2&gt;What kind of a device is it?&lt;/h2&gt;
Please make sure the device is plugged in (or turned on) before proceeding&lt;br&gt;
Anything not listed here should be detected automatically, if supported</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <widget class="QPushButton" name="ButtonDMX">
           <property name="text">
            <string>DMX</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QPushButton" name="ButtonGovee">
           <property name="text">
            <string>Govee</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="ButtonKasaSmart">
           <property name="text">
            <string>Kasa Smart</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="ButtonPhilipsWiz">
           <property name="text">
            <string>Philips Wiz</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QPushButton" name="ButtonQMK">
           <property name="text">
            <string>QMK Device</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="ButtonNanloeaf">
           <property name="text">
            <string>Nanoleaf</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="ButtonElgatoKeyLight">
           <property name="text">
            <string>Elgato KeyLight</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="ButtonE131">
           <property name="text">
            <string>E1.31</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="ButtonLIFX">
           <property name="text">
            <string>LIFX</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="ButtonSerial">
           <property name="text">
            <string>Serial Port Device</string>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="ButtonPhilipsHue">
           <property name="text">
            <string>Philips Hue</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="ButtonElgatoLightStrip">
           <property name="text">
            <string>Elgato LightStrip</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QPushButton" name="ButtonYeelight">
           <property name="text">
            <string>Yeelight</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="PageScan">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="LabelScan">
         <property name="text">
          <string>&lt;h2&gt;Re-scanning devices...&lt;/h2&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="ButtonScanBack">
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonScanRescan">
           <property name="text">
            <string>Scan again</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonScanBeginning">
           <property name="text">
            <string>Add another device</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonScanClose">
           <property name="text">
            <string>All good!</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="PageConfigure">
      <layout class="QVBoxLayout" name="PageConfigureLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>&lt;h2&gt;Please configure your device here&lt;/h2&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="ConfigureEntry" native="true"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="ButtonConfigureBack">
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonConfigureScan">
           <property name="text">
            <string>Save &amp;&amp; Scan</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SlidingStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>slidingstackedwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
